### PR TITLE
Add GitHub oauth scope info

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Explain OAuth scope for GitHub integrations](https://trello.com/c/VNW1qSnI/2026-explain-oauth-scope-for-github-integrations)

File: _console/integrations.md [hunk 1]
```diff
@@ -18,6 +18,24 @@ When enabling the integration, please note that there are OAuth settings for eac
 
 Also note that forks from private repos in unauthorized GitHub organizations will not be available for triggers, even if the fork's current organization is authorized. To work around this limitation, instead of forking you can clone the original repo and push it to a new repo in an authorized organization.
 
+### Write permissions
+
+We are currently requesting `user:email`, `admin:repo_hook` and `repo` scope.
```
Comment (mattmanning) [inline _console/integrations.md]: Please say "OAuth scope" here instead of just "scope".

File: _console/integrations.md [hunk 2]
```diff
@@ -18,6 +18,24 @@ When enabling the integration, please note that there are OAuth settings for eac
...
+
+We request `repo` scope so we can:
+
+- List all your repositories for easy selection in the integration and workflow builder tools
```
Comment (mattmanning) [inline _console/integrations.md]: Please capitalize "Workflow" everywhere here since it's the name of a feature.

File: _console/integrations.md [hunk 3]
```diff
@@ -18,6 +18,24 @@ When enabling the integration, please note that there are OAuth settings for eac
...
+- Query a repo branch to make sure other required status checks pass before running a workflow
+- Pull code to build Docker images during a workflow
+
+Unfortunately, `repo` scope also includes write permissions. Therefore, according to our current understanding of the GitHub OAuth feature, it's not currently possible to grant us read-only access to your repos.
```
Comment (mattmanning) [inline _console/integrations.md]: Let's cut the "Therefore . .  " sentence and just say ", but Convox does not write to your repos."

**Pull request merged**